common/common.cpp

```diff
@@ -995,6 +995,14 @@ std::vector<llama_arg> gpt_params_parser_init(gpt_params & params, llama_example
             params.flash_attn = true;
         }
     ).set_env("LLAMA_ARG_FLASH_ATTN"));
+    add_opt(llama_arg(
+        {"--no-perf"},
+        format("disable internal libllama performance timings (default: %s)", params.no_perf ? "true" : "false"),
+        [](gpt_params & params) {
+            params.no_perf = true;
+            params.sparams.no_perf = true;
+        }
+    ).set_env("LLAMA_ARG_FLASH_ATTN"));
```

Comment:
```suggestion
    ));
```

Hmm I should also add a test for duplicated env var in `test-arg-parser.cpp`